src/models/guild.rs

```diff
@@ -160,3 +161,4 @@ bitflags::bitflags! {
 }
 
 serde_for_bitflags!(u32: GuildFlags);
+bincode_for_bitflags!(GuildFlags);
```

Comment:
such can be combined with serde_for_bitflags!

Reply:
feel free